MzML/MzmlMethods.cs

```diff
@@ -406,109 +656,117 @@ public static void CreateAndWriteMyMzmlWithCalibratedSpectra(IMsDataFile<IMsData
-                        accession = "MS:1000523",
-                        name = "64-bit float"
-                    };
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[3].cvParam[2] = new Generated.CVParamType()
-                    {
-                        accession = "MS:1000576",
-                        name = "no compression"
-                    };
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[3].userParam = new Generated.UserParamType[1];
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[3].userParam[0] = new Generated.UserParamType()
-                    {
-                        name = "kelleherCustomType",
-                        value = "noise baseline"
-                    };
-
-                    // baseline
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[4] = new Generated.BinaryDataArrayType()
-                    {
-                        binary = myMsDataFile.GetOneBasedScan(i).Get64BitNoiseDataBaseline()
-                    };
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[4].encodedLength = (4 * Math.Ceiling(((double)mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[4].binary.Length / 3))).ToString(CultureInfo.InvariantCulture);
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[4].cvParam = new Generated.CVParamType[3];
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[4].cvParam[0] = new Generated.CVParamType()
-                    {
-                        accession = "MS:1000786",
-                        name = "non-standard data array"
-                    };
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[4].cvParam[1] = new Generated.CVParamType()
-                    {
-                        accession = "MS:1000523",
-                        name = "64-bit float"
-                    };
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[4].cvParam[2] = new Generated.CVParamType()
-                    {
-                        accession = "MS:1000576",
-                        name = "no compression"
-                    };
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[4].userParam = new Generated.UserParamType[1];
-                    mzML.run.spectrumList.spectrum[i - 1].binaryDataArrayList.binaryDataArray[4].userParam[0] = new Generated.UserParamType()
-                    {
-                        name = "kelleherCustomType",
-                        value = "noise intensity"
-                    };
-                }
+                //Following seems to be uneeded. keep code for now incase.
+                /*
```

Comment:
At least some users wanted this. Can it be enabled while still working with other software?